File: src/main/java/top/zbeboy/isy/web/vo/data/building/BuildingVo.java
package top.zbeboy.isy.web.vo.data.building;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2017/5/31.
 */
@Data
public class BuildingVo {
    private Integer buildingId;
    @NotNull
    @Size(max = 200)
    private String buildingName;
    private Byte buildingIsDel;
    private Integer collegeId;
}

File: src/main/java/top/zbeboy/isy/web/vo/data/building/BuildingVo.kt
package top.zbeboy.isy.web.vo.data.building

import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-07 .
 **/
open class BuildingVo {
    var buildingId: Int? = null
    @NotNull
    @Size(max = 200)
    var buildingName: String? = null
    var buildingIsDel: Byte? = null
    @NotNull
    @Min(1)
    var collegeId: Int? = null
}